File: Iniciante/1042 - Sort Simples/JavaScript/1042.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split(" ");
  let lista = lines.map((item) => parseInt(item, 10));

  crescente(lista);
  original(lista);
}

function crescente(l) {
  let lista = l.slice();
  lista.sort((a, b) => a - b);

  for (let item of lista) {
    console.log(item);
  }

  console.log();
}

function original(l) {
  for (let item of l) {
    console.log(item);
  }
}

main();

File: Iniciante/1042 - Sort Simples/Python/1042.py
def main():
    entrada = input().split()
    lista = list(map(int, entrada))

    crescente(lista)
    original(lista)


def crescente(l):
    crescente = sorted(l)

    for item in crescente:
        print(item)

    print()


def original(l):
    for item in l:
        print(item)


main()
